src/services/JobRoleService.ts

```diff
@@ -0,0 +1,20 @@
+import axios, { AxiosResponse } from "axios";
+import { JobRoleResponse } from "../models/JobRoleResponse";
+
+axios.defaults.baseURL = process.env.API_URL || "http://localhost:8080/";
+
+export const URL: string = "/api/job-roles";
```

Comment:
what's this?

Reply:
It's used to set the base url used in axios requests we haven't set the prod url yet since we don't have the backend deployed yet so it defaults to localhost, the URL

- I just realized I hard coded the full URL a few lines below anyway which doesn't utilize it but I just made the changes